cmd/zfs/zfs_main.c

```diff
@@ -4399,6 +4455,7 @@ zfs_do_send(int argc, char **argv)
 	int c, err;
 	nvlist_t *dbgnv = NULL;
 	char *redactbook = NULL;
+	struct zfs_send_exclude_arg exclude_context = {};
```

Comment:
```suggestion
	zfs_send_exclude_arg_t exclude_context = { 0 };
```

Should be initialized.